### DISCUSSION TITLE
Can SumatraPDF not open linked MP4s from presentation into a native video app?

### DISCUSSION THREAD

**Discussion by OTJArchitects** [sumatrapdfreader/sumatrapdf#4086] [Q&A]
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/156936847/b63aa794-2b93-4eb9-b571-9c35bb0c90fe)

I feel like this isn't the intended functionality. Is there a way we should be linking the file?

I saw [this issue](https://github.com/sumatrapdfreader/sumatrapdf/issues/618), but I don't know if its entirely relevant unless ,mp4 isn't a safe link type. 

Thanks,
Noah

**Comment by GitHubRulesOK** [comment]
That option is available in some less secure readers like Acrobat was but unsure how much they have locked down in recent releases, certainly they removed SWFlash as too easy to write exploits.

If the file is a link to runnable MP4 (local or remote) current SumatraPDF will NOT by default allow you to be highjacked  by phishing "click here"

If the MP4 is imbedded as it should be as "Rich Media Comment/Annotation" then Acrobat should raise a warning and ask if you trust it
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/15226df9-c54e-494d-a597-8f9c7380b2c5)


**Current SumatraPDF cannot invoke it nor export it, and older versions would only save it, NOT run it for you.**

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/0a61c15f-9d87-4801-86f8-c1b7d62e315b)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/85b59e3a-85ef-4bb6-84c6-b48fec799f7f)

**Comment by kjk** [comment]
Please open an issue and provide a test file.
